Marlin/src/module/tool_change.cpp

```diff
@@ -786,7 +786,9 @@ void tool_change_prime() {
   if (toolchange_settings.extra_prime > 0
     && TERN(PREVENT_COLD_EXTRUSION, !thermalManager.targetTooColdToExtrude(active_extruder), 1)
   ) {
-    destination = current_position; // Remember the old position
+    // For BACKUP_POWER_PARK_WAITING set 'destination' before calling
+    // so that the code below knows the return point and E value.
+    TERN(BACKUP_POWER_PARK_WAITING, /*nothing*/, destination = current_position);
```

Comment:
How does this change affect `M217 Q` and `TOOLCHANGE_FS_PRIME_FIRST_USED` (in `tool_change`)?

Reply:
Toolchange_prime() start from a point of the print , and return to it .... But if called for park ' because power loss first go to park and call prime() at this position ' , it was my idea to give the destination data to prime function

Reply:
And really interest to use the priming sequence for resuming that is really adjusted , no string , no clogs

Reply:
I have a request for you ... But need time and reorganisation .....   Park definition in tool_change , park definition in nozzle_park for advanced pause , park definition in PowerLoss ..... 
In nozzle_park , the class is ready , need just to be universal ... and need to remember the first point to have a function to return